source/features/conversation-authors.tsx

```diff
@@ -28,7 +28,8 @@ async function highlightCollaborators(signal: AbortSignal): Promise<void> {
 		'.js-issue-row [data-hovercard-type="user"]',
 		'a[class^="IssueItem-module__authorCreatedLink"]',
 	], author => {
-		if (list.includes(author.textContent.trim())) {
+		const name = author.textContent.trim();
+		if (list.includes(name) && name !== getUsername()) {
```

Comment:
Note that this also stops highlighting your own username when it's detected as a collab, because I don’t like how this looked (and also because it seems unnecessary)

<img width="643" height="350" alt="Image" src="https://github.com/user-attachments/assets/7692f282-2028-4cf0-9f7b-8a04b1ccb5e5" />